fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/AggregatedMaterializedViewRewriter.java

```diff
@@ -60,6 +62,8 @@ public class AggregatedMaterializedViewRewriter extends MaterializedViewRewriter
             .put(FunctionSet.COUNT, FunctionSet.SUM)
             .build();
 
+    private static Set<String> UNSUPPORTED_FUNCTIONS = ImmutableSet.<String>builder().add(FunctionSet.AVG).build();
```

Comment:
What does unsupported mean ?

Reply:
unsupported rollup. I will change it to UNSUPPORTED_ROLLUP_FUNCTIONS